### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by shreyanshu28** [shreyanshu28/text-analysis#5]
from sklearn.metrics import confusion_matrix
import seaborn as sns
y_pred = clf.predict(X_test)
conf = metrics.confusion_matrix(y_test, y_pred)
fig, ax = plt.subplots(figsize=(10, 10))
sns.heatmap(conf, annot=True, fmt='d')

plt.ylabel('True Label')
plt.xlabel('Predicted Label')
plt.show()
le.fit(df['Product'])
le_name_mapping1 = dict(zip(le.classes_, le.transform(le.classes_)))
print(le_name_mapping1) # maps product to ID
le_name_mapping2 = dict(zip(le.transform(le.classes_),le.classes_))
print(le_name_mapping2) # maps product to ID
